Add some parameters to push-to-docker (#205)
giantswarm/architect-orb: docs/job/push-to-docker.md

# push-to-docker

This job builds a docker image and pushes it to a registry.
It uses the Dockerfile at the root of the workspace directory and the root directory as build context by default.

> Otherwise, it is possible to specify the used Dockerfile and build context by specifying `dockerfile` and `build-context` arguments.

Reviewer: ```suggestion
Otherwise, it is possible to specify the Dockerfile and build context to use with `dockerfile` and `build-context` arguments respectively.
```

          image: "quay.io/giantswarm/REPOSITORY"
          username_envar: "QUAY_USERNAME"
          password_envar: "QUAY_PASSWORD"

>           build-context: "."

Reviewer 2: Maybe `build-path`?

```
$ docker build --help | head -n 2

Usage:  docker build [OPTIONS] PATH | URL | -
```

Author: Docker calls this `context` so if you don't have a strong preference to call this `build-path` I think `build-context` is better.
